Doc update (#166)
Scalr/terraform-provider-scalr: docs/data-sources/scalr_policy_group.md

* `id` - An identifier of the policy group.
* `status` - A system status of the policy group.
* `error_message` - An error details if Scalr failed to process the policy group.

> * `opa_version` -  A version of Open Policy Agent the policy group verifies run on. 

Reviewer: ```suggestion
* `opa_version` -  The version of the Open Policy Agent that the policy group is using. 
```

> * `vcs_provider_id` - An identifier of a VCS provider in the policy group is created. In the format `vcs-<RANDOM STRING>`.

Reviewer: ```suggestion
* `vcs_provider_id` - The VCS provider idenitfier for the repository where the policy group resides. In the format `vcs-<RANDOM STRING>`.
```

> * `vcs_repo` - Contains VCS-related meta-data the policy group created from.

Reviewer: ```suggestion
* `vcs_repo` - Contains VCS-related meta-data for the policy group.
```
